packages/create-next-app/index.ts

```diff
@@ -169,6 +169,7 @@ const program = new Commander.Command(packageJson.name)
   )
   .allowUnknownOption()
   .parse(process.argv)
+  .opts()
```

Comment:
@torresgol10 Can you also clarify what the reason was for `.opts()` <picture data-single-emoji=":thinkies:" title=":thinkies:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thinkies/d164ba479fed9c0d.png" alt=":thinkies:" align="absmiddle"></picture>

Reply:
@samcx  If I remove that call, typescript errors out every time `program.XXX` is made so that command returns an object. 

I attach a screenshot of the official commander documentation: ![image](https://github.com/vercel/next.js/assets/40922354/c3d5be92-fe7b-4b6d-a292-7573b30c2e52)

Reply:
@torresgol10 Noted. I think we want to refactor `create-next-app`'s code to be more similar with the Next.js CLI code (arguably more readable, e.g., not doing `process.argv.includes`), but for another time to take a look at!